conote-frontend/src/components/editor/EditorNavbar.tsx

```diff
@@ -471,7 +475,7 @@ export default function EditorNavbar({
       <HStack spacing="4">
         <HStack align="center" spacing="1">
           <UserPresenceAvatars userPresence={userPresence} />
-          {userRole !== "viewer" && (
+          {userRole !== undefined && userRole !== "viewer" && (
```

Comment:
Perhaps can make it so all the right-hand side components of the navbar don't render whilst user role is undefined, because undefined user roles imply users don't have access to the document (either loading or prohibited), this way it'll prevent toast messages from appearing.